dist/react-formbuilder.js

```diff
@@ -1331,26 +1331,91 @@ module.exports = React.createClass({
 "use strict";
 
 
-var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
-
-var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; };
-
 var React = __webpack_require__(0);
 var cleanProps = __webpack_require__(1);
 var MultiplicityField = __webpack_require__(5);
 
 module.exports = React.createClass({
   displayName: "exports",
+  getInitialState: function getInitialState() {
+    console.log(this.props);
+    return {
+      attachment: false
+    };
+  },
   render: function render() {
+    var _this = this;
+
     var props = this.props;
-    var value = props.value;
+    var className = (this.props.className || '') + ' image';
 
-    if (props.multiplicity) {
-      var values = (typeof value === "undefined" ? "undefined" : _typeof(value)) === "object" ? value : [value];
-      return React.createElement(MultiplicityField, _extends({}, props, { values: values }));
+    return React.createElement(
+      "div",
+      { className: className },
+      React.createElement("input", { type: "file", hidden: "hidden", ref: "optionalFile", onChange: function onChange(e) {
+          return _this.handleFiles(e);
+        } }),
+      this.generatePicker()
+    );
+  },
+
+
+  generatePicker: function generatePicker() {
+    var _this2 = this;
+
+    if (!this.state.attachment) {
+      return React.createElement("input", { type: "button", className: "btn attach", onClick: function onClick(e) {
+          return _this2.selectFiles(e);
+        }, value: "Click here to pick an image" });
     }
 
-    return React.createElement("input", _extends({ type: "text" }, cleanProps(props)));
+    return [React.createElement("img", { key: "preview", src: "data:image/jpg;base64," + this.state.attachment.base64 }), React.createElement("input", { key: "attach", type: "button", className: "btn reattach", onClick: function onClick(e) {
+        return _this2.selectFiles(e);
+      }, value: "Click here to pick a different image" })];
+  },
+
+  selectFiles: function selectFiles() {
+    this.refs.optionalFile.click();
```

Comment:
Any reason you're hiding the native input and using a button instead, when you could just change the value of the input? Thinking about possible benefits of using the native image input element (drag and drop capability? Accessibility?)